lang/java/avro/src/main/java/org/apache/avro/reflect/AvroAliases.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.avro.reflect;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
```

Comment:
nit: can you please move the license to the beginning of this file, above package?

Reply:
@baunz Can you pick this up? This should also be caught by the RAT check.

Reply:
I ran RAT check, but looks like it didn't catch this, is it searching in the entire file? Anyway, I think better to have the license at the beginning of the file, at least it looks better. Otherwise this PR seems to a good addition!

Reply:
Sure to fix this, I just needed to return to my home pc. I also think it should be auto-checked or even auto inserted via a license header step in spotless. For now i will manually insert it